examples/use-cases/vuln_sync_csv_upload/vuln_upload.py

```diff
@@ -0,0 +1,197 @@
+"""Use case for automating the ingestion of CVE reports"""
+import os
+import sys
+import time
+from getpass import getpass
+from io import FileIO
+
+import requests
+
+
+def upload_cve(file: FileIO):
+    """ Uploads vulnerability list to Automox Vulnerability Sync endpoint.
+
+    Args:
+        file (FileIO): A CSV file containing vulnerability data.
+
+    Returns:
+        response_data (dict): API response from Automox Vulnerability Sync
+        https://developer.automox.com/openapi/vulnsync/operation/UploadCSVBatch/
+    """
+
+    response_data = {}
+
+    task = "patch"
+    url  = f"https://console.automox.com/api/orgs/{organization}/tasks/{task}/batches/upload"
+
+    filename = os.path.basename(file.name)
+
+    try:
+        headers = {
+            "Authorization": f"Bearer {api_secret}",
+        }
+
+        files = [
+            ('file', (filename, file, 'text/csv'))
+        ]
+
+        response = requests.request("POST", url, headers=headers, files=files)
+
+        response_data = response.json()
+
+        if "errors" in response_data and len(response_data['errors']) > 0:
+            msg = ""
+            msg = msg.join(response_data['errors'])
+
+            raise Exception(msg)
+    except requests.RequestException as error:
+        print(f"Error: Unable to complete CSV upload request. ({error})")
+
+    return response_data
+
+def get_unprocessed_cves(directory: str):
+    """Returns a list of CSV files to upload and process.
+
+    Args:
+        directory (str): Directory to look in for CSVs.
+
+    Returns:
+        cve_files (list): List of files to be processed and uploaded.
+    """
+
+    cve_files = []
+
+    paths = os.listdir(directory)
```

Comment:
You might be able to clean up the loop a bit by using `glob`: https://docs.python.org/3/library/glob.html